lib/fs.js

```diff
@@ -1209,7 +1209,7 @@ function toUnixTimestamp(time) {
   }
   throw new errors.Error('ERR_INVALID_ARG_TYPE',
                          'time',
-                         ['Date', 'time in seconds'],
+                         ['Date', 'Time in seconds'],
```

Comment:
Not really need to be fixed in this PR, but this should be `['Date', 'string', 'number']`